lab-sarah/ng-template/app/view/home/home-controller.js

```diff
@@ -0,0 +1,41 @@
+'use strict';
+
+require('./_home.scss');
+
+module.exports = ['$log', '$rootScope', 'galleryService', HomeController ];
+
+function HomeController($log, $rootScope, galleryService){
+  $log.debug('init homeCtrl');
+  //an array of gallery ID's, shares same reference as array in gallery-service
+  this.galleries = [];
+
+  //when you call fetchGalleries on the gallery Service, reassigns the gallery array to the array in the gallery-service. So when you change the gallery array, it changes in the service and the home-controller. pass by reference
+  this.fetchGalleries = function() {
+    galleryService.fetchGalleries()
+    .then(galleries => {
+      this.galleries = galleries;
```

Comment:
Good job storing a reference to the galleries array on your service on the scope of your home controller!